crates/turbopack-node/src/pool.rs

```diff
@@ -167,7 +234,16 @@ impl NodeJsPoolProcess {
                         .unwrap()
                         .read_to_end(&mut stderr)
                         .await?;
-                    Ok((String::from_utf8(stdout)?, String::from_utf8(stderr)?))
+                    fn clean(buffer: Vec<u8>) -> Result<String> {
+                        Ok(String::from_utf8(buffer)?
+                            .lines()
+                            .filter(|line| {
+                                line.len() != MARKER_STR.len() + 1 && !line.starts_with(MARKER_STR)
```

Comment:
Why the `line.len() != MARKER_STR.len() + 1 ` condition?

Reply:
Fast check for the right len